crates/swc_ecma_transforms_optimization/src/simplify/dce/mod.rs

```diff
@@ -30,6 +30,7 @@ use crate::debug_assert_valid;
 pub fn dce(
     config: Config,
     unresolved_mark: Mark,
+    drop_unused_imports: bool,
```

Comment:
This is a breaking change anyway

Reply:
This is exactly... Sorry for your confusion 🙇‍♂️ 
I updated the PR comment.